### PR TITLE
Feat/docker

### PR DESCRIPTION
**Pull request by Huskydog9988**

### CODE REVIEW

File: apps/web/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
+FROM node:18-alpine AS builder
+# ensure deps can build
+RUN apk add --update --no-cache openssl1.1-compat libc6-compat
+# enable pnpm
```
Comment (ceriddenn) [inline apps/web/Dockerfile:4]: are we still using pnpm? or is this old..

File: apps/web/next.config.js [hunk 2]
```diff
@@ -1,7 +1,19 @@
+const path = require('path');
+
```
Comment (ceriddenn) [inline apps/web/next.config.js:2]: make sure when we merge this, the security headers get added.

File: apps/web/Dockerfile (hunk 1 above)
Comment (Huskydog9988) [reply apps/web/Dockerfile]: we still need it for yarn, i just fixed the comments

**Review by ceriddenn — APPROVED** [review]

**Review by ceriddenn — APPROVED** [review]

File: apps/backend/package.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
   "keywords": [],
   "dependencies": {
     "axios": "^1.1.3",
-    "database": "*",
+    "database": "workspace:*",
```
Comment (tsundDEV) [inline apps/backend/package.json:16]: Fix `workspace:*` to `*`

**Review by tsundDEV — APPROVED** [review]

**Pull request merged**